Makefile

```diff
@@ -27,7 +27,7 @@ build-debug: install-tools
 
 .PHONY: run
 run: build
-	$(CUSTOM_COL_DIR)/grafana-ci-otelcol --config config/config.yaml
+	$(CUSTOM_COL_DIR)/grafana-ci-otelcol --config config.yaml
```

Comment:
why do we need this change?

Reply:
when you run `make run` from the root of the project `config.yaml` is expected to be in the root directory, not in the config directory (same level of `config.example.yaml`)

it's also mentioned in the readme:

```
## Configuration

The collector is configured using the `config.yaml` file in the root of the repository.
You can learn more about the OpenTelemetry Collector configuration [here][otel-configuration].
```

this was probably a leftover form when i did the big refactoring, i didn't catch it earlier because i had a bunch on unstaged and uncommitted files around

Reply:
okay - does [this](https://github.com/grafana/deployment_tools/blob/master/ksonnet/lib/cicd-o11y/cicd-o11y.libsonnet) need to change as well?

Reply:
no, there we specify the config path as an argument
```
+ container.withArgs([
        '--config',
        '/config/config.yaml',
      ])
```

because we mount the config file in that directory. `make run` is only used locally for development